src/thriveopedia/pages/wiki/ThriveopediaOrganellesRootPage.tscn

```diff
@@ -0,0 +1,20 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://src/thriveopedia/ThriveopediaPage.tscn" type="PackedScene" id=1]
+[ext_resource path="res://src/thriveopedia/pages/wiki/ThriveopediaOrganellesRootPage.cs" type="Script" id=2]
+
+[node name="ThriveopediaPage" instance=ExtResource( 1 )]
+script = ExtResource( 2 )
+MainArticlePath = NodePath("MarginContainer/ScrollContainer/MarginContainer/VBoxContainer/VBoxContainer")
+OrganelleListContainerPath = NodePath("MarginContainer/ScrollContainer/MarginContainer/VBoxContainer/GridContainer")
+
+[node name="VBoxContainer" type="VBoxContainer" parent="MarginContainer/ScrollContainer/MarginContainer/VBoxContainer" index="0"]
+margin_right = 864.0
+
+[node name="GridContainer" type="GridContainer" parent="MarginContainer/ScrollContainer/MarginContainer/VBoxContainer" index="1"]
```

Comment:
Would an HFlow container work here? That would allow the columns to appear flexibly, so it might be better in some situations. I haven't tested this yet so I don't know if this suggestion is better or not.

Reply:
Good point, changed and it works much better.